docs/integration_guide.md

```diff
@@ -26,31 +26,130 @@ The order of events for relaying transactions or deploying smart wallets through
 Another option is to use Enveloping through a Relay Provider. The latter wraps web3, and then all transactions and calls are made through the Relay Provider. If a Relay Client is not provided then the Relay Provider creates an instance.
 
 ```typescript
-    this.config = await resolveConfiguration(web3.currentProvider, {
-      verbose: window.location.href.includes('verbose'),
-      onlyPreferredRelays: false, //If false it will look for a relayer, if true it reads preferred Relays
-      chainId: chainId,
-      relayVerifierAddress: this.contracts.relayVerifier,
-      factory: this.contracts.factory,
-      preferredRelays:[]
-    })
-
-    this.provider = new Enveloping.RelayProvider(web3.currentProvider, this.config)
-    web3.setProvider(this.provider)
-
-    this.provider.deploySmartWallet(trxData)
-    this.provider.calculateSmartWalletAddress(
-    factory.address,gaslessAccount.address, recoverer, customLogic, walletIndex, bytecodeHash)
-
-    await testRecipient.emitMessage('hello world', {
-        from: gaslessAccount.address,
-        gas: '100000',
-        gasPrice: '1',
-        callVerifier: verifierInstance.address,
-        onlyPreferredRelays: true //It will read the preferredRelays on the config.
-    })
+    import { RelayProvider, resolveConfiguration } from "@rsksmart/enveloping";
+    import Web3 from "web3";
+
+    const ZERO_ADDRESS = "0x0000000000000000000000000000000000000000";
+
+    const web3 = new Web3("http://localhost:4444");
+    
+    const smartWalletFactoryAbi = {};// some json containing the abi of the smart wallet factory contract.
+    const smartWalletFactoryAddress = "0x3bA95e1cccd397b5124BcdCC5bf0952114E6A701"; // the smart wallet factort contract address (can be retrieved from the summary of the deployment).
```

Comment:
(valid for the rest of the doc)

Reply:
Is not, but it will be, and also the actual output of the deployment shows you the addresses of the contracts so is not to far of a summary. Do you still want to change that comment?

Reply:
nope, with what you're saying in mind, i think it's fine as it is :+1: